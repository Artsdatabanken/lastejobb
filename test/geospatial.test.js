const { geospatial } = require("..");

const geom = [[0, 0], [0, 10], [10, 10], [0, 0]];
const geom2 = [[50, 0], [0, 1], [10, 1], [0, 0]];

test("area", () => {
  const actual = geospatial.calculateArea(geom);
  expect(actual).toMatchSnapshot();
});

test("bbox", () => {
  const actual = geospatial.axisAlignedBoundingBox(geom);
  expect(actual).toMatchSnapshot();
});

test("bbox multi", () => {
  const actual = geospatial.axisAlignedBoundingBox([geom, geom2]);
  expect(actual).toMatchSnapshot();
});
