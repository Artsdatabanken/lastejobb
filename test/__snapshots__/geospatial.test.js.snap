// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`area 1`] = `50`;

exports[`bbox 1`] = `
Array [
  0,
  0,
  10,
  10,
]
`;

exports[`bbox multi 1`] = `
Array [
  0,
  0,
  50,
  10,
]
`;
